src/sidebar/services/annotation-activity.js

```diff
@@ -1,3 +1,4 @@
+import { isShared as _isShared } from '../helpers/permissions';
```

Comment:
This alias isn't strictly required. `isShared: isShared(annotation.permissions)` below will work.